components/aws/sagemaker/Changelog.md

```diff
@@ -4,6 +4,13 @@ The version of the AWS SageMaker Components is determined by the docker image ta
 Repository: [Public ECR](https://gallery.ecr.aws/kubeflow-on-aws/aws-sagemaker-kfp-components) or [Dockerhub](https://hub.docker.com/repository/docker/amazon/aws-sagemaker-kfp-components). New releases after v1.1.1 will be using the public ECR repository
 
 ---------------------------------------------
+** Change log for version 2.2.1 **
+- Fixing Path bug when cloning repository on macbook
```

Comment:
Add a line - Directory name for version 2 of Model component has been updated from `Model` to `Modelv2`

Reply:
Fix: Path issue with git clone on macbook. Directory name for version 2 of Model component has been updated from `Model` to `Modelv2`